### ISSUE TITLE
Hovering event is not working

### ISSUE THREAD

**Issue by saadaoui-3** [meslzy/electron-as-wallpaper#26]
I want to enable some animations on hover, i test the window before attach, it's working fine
but when using attack, the hover event is not triggered till i click on something like how hover works in phones

**Comment by saadaoui-3** [comment]
@meslzy, i forgot to assign you in the issue, so i had to mention you here 🙌

**Comment by saadaoui-3** [comment]
https://github.com/user-attachments/assets/c5b30f78-51eb-4a7c-8159-6250d96a9ce4

**Comment by meslzy** [comment]
Hello @saadaoui-3, i have tried to solve the issue, but i could not succeed. 
Probably this issue is beyond my expertise, but ill try my best ill keep the issue pinned and active.

**Comment by saadaoui-3** [comment]
Hi @meslzy, good luck! your effort to learn and improve is truly admirable!

**Comment by saadaoui-3** [comment]
Hi @meslzy,

The issue seems to have resolved, though I’m unsure of the exact reason. I made several unrelated changes in my wallpaper app which might have contributed indirectly, but i am not sure.

The only main change i did, was moving electron from dependencies to devDependencies. but i am not sure if this is the solution

```
	"dependencies": {
		"electron-as-wallpaper": "^2.0.2",
		"ws": "^8.18.0"
	},
	"devDependencies": {
		"electron": "^33.2.1",
		"electron-builder": "^25.1.8",
		"electron-reload": "^2.0.0-alpha.1"
	}
```

https://github.com/user-attachments/assets/c304867e-eaf8-4717-bb7e-6724a56f5db8

**Comment by saadaoui-3** [comment]
@meslzy, No, it’s not resolved yet. I just noticed that in the orange wallpaper example above, they didn’t rely on hover styling. Instead, they used event listeners to track mouse movements, capturing its current coordinates to dynamically adjust the wallpaper’s styling using JavaScript rather than CSS.

That said, this issue can be resolved by making a small adjustment to the WebBrowser content itself not is `attach` function, as the first counter example isn’t a realistic wallpaper—nobody would embed an entire website on their desktop. This issue can be closed since it can be addressed differently using the mouse event listeners.

**Comment by saadaoui-3** [comment]
I created this for anyone who comes across the same question and wants to start using event listeners. Of course, I’ve tested it using the `attach` function in the video example linked below.

```
<!DOCTYPE html>
<html lang="en">
	<head>
		<meta charset="UTF-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
		<title>Mouse is Working</title>
		<style>
			body {
				background-color: black;
				color: white;
				height: 100vh;
				margin: 0;
				overflow: hidden;
				position: relative;
			}
			.dot {
				position: absolute;
				width: 2px;
				height: 2px;
				background-color: white;
			}
		</style>
	</head>
	<body>
		<script>
			document.addEventListener('mousemove', (event) => {
				const dot = document.createElement('div');
				dot.className = 'dot';
				dot.style.left = `${event.clientX}px`;
				dot.style.top = `${event.clientY}px`;
				document.body.appendChild(dot);
			});
		</script>
	</body>
</html>
```
Here’s the video link for reference:

https://github.com/user-attachments/assets/66ffb704-0dbb-48bd-89c8-f022708027ac
